### PR TITLE
Auto select song book if theres only 1 songbook

### PR DESCRIPTION
**Pull request by brandonxia01**
The "choose a song book" page is kind of a useless step if theres only 1 book. Putting this in place to just auto select the only book if theres only 1, and it will auto update when Red Book gets finished.

### CODE REVIEW

**Review by 123asian — CHANGES_REQUESTED** [review]
Since you got rid of line 43, defaultNavigationTitle isn't used anywhere, so we can probably remove the import in line 2

File: src/pages/HomePage.tsx [hunk 1]
```diff
@@ -17,6 +17,9 @@ const HomePage: React.FC = () => {
 
   useEffect(() => {
     getSongbooks().then((books) => {
+      if (books.length == 1) {
```
Comment (MiTo0o) [inline src/pages/HomePage.tsx:20]: is this meant to be ==?

Comment (brandonxia01) [reply src/pages/HomePage.tsx:20]: mm probably `===` is right, ty

**Comment by brandonxia01** [comment]
> Since you got rid of line 43, defaultNavigationTitle isn't used anywhere, so we can probably remove the import in line 2

Good catch

**Comment by brandonxia01** [comment]
@123asian @MiTo0o can you re review

**Review by brotatotes — APPROVED** [review]

**Pull request merged**